receivers/pagerduty/config.go

```diff
@@ -33,7 +44,7 @@ var getHostname = func() (string, error) {
 type Config struct {
 	Key           string            `json:"integrationKey,omitempty" yaml:"integrationKey,omitempty"`
 	Severity      string            `json:"severity,omitempty" yaml:"severity,omitempty"`
-	CustomDetails map[string]string `json:"-" yaml:"-"` // TODO support the settings in the config
+	CustomDetails map[string]string `json:"custom_details,omitempty" yaml:"custom_details,omitempty"`
```

Comment:
let's rename it to `Details` to match the upstream. https://github.com/prometheus/alertmanager/blob/41eb1213bb1c7ce0aa9e6464e297976d9c81cfe5/config/notifiers.go#L284

Reply:
including json field name

Reply:
D you think that's a good idea?
- In the docs I linked to in the description, the field is called "Custom Details". There's no mention of a "Details" field.
- The JSON tag for this field is also `"custom_details"`.
- Upstream converts this `"details"` field into `"custom_details"` when it has to send a notification, so it adds an extra step.

https://github.com/prometheus/alertmanager/blob/14a265b71024b858dd8186bd6ace3f666f1e934c/notify/pagerduty/pagerduty.go#L249

Reply:
Yes, our main target is that our config would be as close as possible to upstream Alertmanager config.

Reply:
That makes sense. I'll try to be as close to upstream as possible in my next commit.